### DISCUSSION TITLE
python  深度學習

### DISCUSSION THREAD

**Discussion by Jerry10413** [1101-GS4719-C/Python-Community-GS4719-C#956] [General]
https://windysavage.github.io/how-to-learn-python/
老師在課程教的python的網路爬蟲
其實python也很常拿來做機器學習和深度學習
這次分享的連結就是基礎從零開始學深度學習的文章
希望對各位有幫助

**Comment by peggy0329** [comment]
這篇文章不只整理了線上、線下資源，還有推薦的建置環境！
很適合初學者

**Comment by ghost** [comment]
這篇文章整理了線上、線下資源，方便初學者找尋資料，降低學習難度，但若沒有足夠的毅力，或被逼迫的壓力，縱使有再多資源，也是毫無用處

**Comment by dinosaur1009** [comment]
這篇文章讓初學者也能輕鬆掌握初階的深度學習！十分具有參考價值

**Comment by KH1126** [comment]
最近想要學習深度學習的觀念，
因此有在yt找尋相關教學。
我認為李宏毅老師的機械學習影片蠻淺顯易懂的，
很適合當入門深度學習的影片來學習。

**Comment by YifenLin** [comment]
我之前在看一本有關神經網路的書的時候，有搭配李宏毅的OCW一起學習，覺得老師教的很好，讓我對於神經網路的概念更加理解。這邊分享我當時在看的電子書：http://neuralnetworksanddeeplearning.com/index.html?fbclid=IwAR3GtheU19oT_I0URhnVDbFifEgSxxmbUO8av4eiDVRlIRkhqAvKTXma4js

**Comment by wanderer657** [comment]
非常感謝分享這則文章，這篇文章介紹了從零開始學習程式語言到深度學習的完整過程。作者分享了自己從程式新手到能夠利用Python找到工作的心路歷程，並提供了豐富的學習資源和建議。通過Google搜尋和線上資源，以及李宏毅老師等知名教學的課程，作者逐步掌握了編程技能和深度學習知識。文章給予初學者鼓勵，指明了學習的方向，我認為對於想進入機器學習領域的人提供了有價值的參考和指導。

**Comment by BobomanOUOb** [comment]
我自己之前在做實驗的時候用過OpenCV搭配AI的Deep Learning 來利用AI來追蹤物體，我之前和學長研究了一段時間，現在看完這篇文章，他對於初學者來說寫的很明瞭，很適合當初學者的入門文章

**Comment by jeff110303549** [comment]
這篇文章看起來是一個很好的入門指南，對於想要從零開始學習深度學習的人來說應該會很有幫助。深度學習是當今人工智慧領域的一個重要分支，掌握深度學習技術可以讓人們在各種應用領域中取得更好的成果。感謝你分享這個連結！
